### ISSUE TITLE
NVAE: A Deep Hierarchical Variational Autoencoder

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#92]
# TL;DR
This is a study of hierarchical VAE to generate high-definition images. 	Instead of having the variance and average calculated directly at each stage, they design the distribution taking into account the relative average of the previous layer. 	In addition, there are various innovations such as swish activation, cells with SE modules, spectral norm, and depth-wise conv for wide receptive field with low calculation cost.
![image](https://user-images.githubusercontent.com/24524018/92561501-46d84a80-f2af-11ea-941f-f8eca078ccf4.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2007.03898 

# Submission Dates(yyyy/mm/dd)

# Authors and institutions

# Methods

# Results

# Comments
